integrations/YandexMetrica/browser.js

```diff
@@ -0,0 +1,150 @@
+import logger from "../../utils/logUtil";
+
+import {
+  ecommEventPayload,
+  sendEvent,
+  commonEventPayload,
+  eventMapping,
+} from "./utils";
+import {
+  removeUndefinedAndNullValues,
+  getHashFromArrayWithDuplicate,
+} from "../utils/commonUtils";
+import { NAME } from "./constants";
+
+class YandexMetrica {
+  constructor(config) {
+    this.tagId = config.tagId;
+    this.clickMap = config.clickMap;
+    this.trackLinks = config.trackLinks;
+    this.trackBounce = config.trackBounce;
+    this.webvisor = config.webvisor;
+    this.containerName = config.containerName;
+    this.eventNameToYandexEvent = config.eventNameToYandexEvent;
+    this.name = NAME;
+  }
+
+  loadScript() {
+    (function (m, e, t, r, i, k, a) {
+      m[i] =
+        m[i] ||
+        function () {
+          (m[i].a = m[i].a || []).push(arguments);
+        };
+      m[i].l = 1 * new Date();
+      for (var j = 0; j < document.scripts.length; j++) {
+        if (document.scripts[j].src === r) {
+          return;
+        }
+      }
+      (k = e.createElement(t)),
+        (a = e.getElementsByTagName(t)[0]),
+        (k.async = 1),
```

Comment:
Add this:
import { LOAD_ORIGIN } from '../ScriptLoader';
k.setAttribute('data-loader', LOAD_ORIGIN),

Reply:
done